deploy/cdk_exec_policy/cdkExecPolicy.yaml

```diff
@@ -0,0 +1,180 @@
+            Action:
+              - 'lambda:AddPermission'
+              - 'lambda:CreateFunction'
+              - 'lambda:GetFunction'
+              - 'lambda:GetFunctionCodeSigningConfig'
+              - 'lambda:GetRuntimeManagementConfig'
+              - 'lambda:PutFunctionEventInvokeConfig'
+              - 'lambda:InvokeFunction'
+            Effect: Allow
+            Resource: '*'
+          - Sid: LambdaPublishLayer
+            Effect: Allow
+            Action:
+              - 'lambda:PublishLayerVersion'
+            Resource:
+              - !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:layer:*'
+          - Sid: S3
+            Action:
+              - 's3:CreateBucket'
+              - 's3:GetBucketPolicy'
+              - 's3:PutBucketCORS'
+              - 's3:PutBucketLogging'
+              - 's3:PutBucketPolicy'
+              - 's3:PutBucketPublicAccessBlock'
+              - 's3:PutBucketTagging'
+              - 's3:PutBucketVersioning'
+              - 's3:PutEncryptionConfiguration'
+              - 's3:PutLifecycleConfiguration'
+            Effect: Allow
+            Resource: 'arn:aws:s3:::*'
+          - Sid: SQS
+            Effect: Allow
+            Action:
+              - 'sqs:CreateQueue'
+              - 'sqs:SetQueueAttributes'
+            Resource: !Sub 'arn:aws:sqs:${AWS::Region}:${AWS::AccountId}:*'
+          - Sid: SSM
+            Effect: Allow
+            Action:
+              - 'ssm:GetParameters'
+              - 'ssm:PutParameter'
+            Resource: '*'
+          - Sid: Logs
+            Effect: Allow
+            Action:
+              - 'logs:CreateLogGroup'
+              - 'logs:CreateLogStream'
+              - 'logs:PutLogEvents'
+              - 'logs:DescribeLogStreams'
+            Resource: 'arn:aws:logs:*:*:*'
+          - Sid: STS
+            Effect: Allow
+            Action:
+              - 'sts:AssumeRole'
+              - 'iam:*Role*'
+            Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:role/cdk-*'
+          - Sid: CloudFormation
+            Effect: Allow
+            Action:
+              - 'cloudformation:*'
+            Resource: !Sub 'arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/CDKToolkit/*'
+          - Sid: ECR
+            Effect: Allow
+            Action:
+              - 'ecr:SetRepositoryPolicy'
+              - 'ecr:GetLifecyclePolicy'
+              - 'ecr:PutImageScanningConfiguration'
+              - 'ecr:DescribeRepositories'
+              - 'ecr:CreateRepository'
+              - 'ecr:DeleteRepository'
+            Resource: !Sub 'arn:aws:ecr:${AWS::Region}:${AWS::AccountId}:repository/cdk-*'
+          - Sid: SSMTwo
+            Effect: Allow
+            Action:
+              - 'ssm:GetParameter'
+              - 'ssm:PutParameter'
+              - 'ssm:DeleteParameter'
+            Resource: !Sub 'arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/cdk-bootstrap/*'
+          - Sid: CloudformationTwo
```

Comment:
For the `CloudfromationTwo` Statement - if my understanding is correct it appears that this statement giving the execution role access to ALL actions as long as the role is being used by CloudFormation

If so is this not the same as the default behavior of adding AdministratorAccess policy to the CDK Execution role? Can we possible restrict this further?